mtg/json.go

```diff
@@ -0,0 +1,38 @@
+package mtg
+
+import (
+	"encoding/json"
+)
+
+// RaritiesToCards maps card rarities to lists of Cards. This map is used for generating packs.
+// This map holds all cards that we can possibly generate. This should be regarded as a temporary
+// solution, since we probably don't want to keep tens of thousands of cards in memory.
+var RaritiesToCards = make(map[CardRarity][]*Card)
+
+// ReadCardData encodes JSON data from a byte array from a single set into a Set struct.
+// The Cards from the Set are stored in the RaritiesToCards global var.
+func ReadCardData(contents []byte) {
+	var s Set
+	_ = json.Unmarshal(contents, &s)
+	// TODO(flp): better error handling
```

Comment:
You could change the signature of `ReadCardData` to be:

```
ReadCardData(contents []byte) error
```

and bail out to return the error early if you encounter a problem